postprocessing-emissions/src/main/java/org/matsim/viz/postprocessing/emissions/VisualizationResource.java

```diff
@@ -29,9 +30,37 @@
     @Path("/data")
     @Produces(MediaType.APPLICATION_JSON)
     @UnitOfWork
-    public String data(@Auth Agent agent, @PathParam("id") String vizId) {
+    public String data(@Auth Agent agent,
+                       @PathParam("id") String vizId,
+                       @QueryParam("startTime") String startTime) {
+
+        QBin binTable = QBin.bin;
+        val bin = new JPAQueryFactory(emFactory.createEntityManager()).selectFrom(binTable)
+                .where(binTable.visualization.id.eq(vizId)
+                        .and(binTable.startTime.eq(parseDouble(startTime))))
+                .fetchFirst();
+
+        if (bin == null) throw new InvalidInputException("Could not find startTime " + startTime);
+
+        return bin.getData();
+    }
+
+    @GET
+    @Path("/bins")
```

Comment:
Maybe it would be a good idea to rename the endpoints.

data -> bin
bins -> startTimes